azure-communication-ui/call-with-chat/build.gradle

```diff
@@ -7,7 +7,7 @@ android {
     compileSdk 31
 
     defaultConfig {
-        minSdk 21
+        minSdk 23
```

Comment:
We also need to update README.md, changelog.md as it is a braking change, and probably a quickstart as well